### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install prereq
+      run: |
+        brew install docker tfenv tflint
+        tfenv install
```
Comment (sblack4) [inline .github/workflows/check.yml:13]: I think this is failing because tfenv is looking for a `.terraform_version` file

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,69 @@
+data "aws_caller_identity" "current" {
+}
```
Comment (sblack4) [inline main.tf:2]: This dangling bracket will never fail to irritate me

**Review by sblack4 — APPROVED** [review]

File: .github/workflows/check.yml (hunk 1 above)
Comment (cdaniluk) [reply .github/workflows/check.yml:13]: added

File: main.tf (hunk 2 above)
Comment (cdaniluk) [reply main.tf:2]: submit a pr for tf fmt. I'm sure they'll accept it!

**Pull request merged**